### PR TITLE
new deployment style: add default env vars & main service key to service definition

### PR DESCRIPTION
**Pull request by ilgooz**
depends on https://github.com/mesg-foundation/engine/pull/1014
resolves https://github.com/mesg-foundation/engine/issues/1022

### CODE REVIEW

**Comment by ilgooz** [comment]
@NicolasMahe feel free to merge this one to dependency PR if you like

File: sdk/service/service.go [hunk 1]
```diff
@@ -85,5 +85,8 @@ func (s *Service) Create(srv *service.Service) error {
 		srv.Sid = "_" + srv.Hash
 	}
 
+	// apply main service key to service.
+	srv.Configuration.Key = service.MainServiceKey
```
Comment (NicolasMahe) [inline sdk/service/service.go]: Why doing this here? It should be in done `server/grpc/api/service.go` in function `fromProtoConfiguration`.

File: server/grpc/api/service.go [hunk 2]
```diff
@@ -73,6 +73,7 @@ func fromProtoConfiguration(configuration *definition.Configuration) *service.De
 		Ports:       configuration.Ports,
 		Volumes:     configuration.Volumes,
 		VolumesFrom: configuration.VolumesFrom,
+		Env:         configuration.Env,
```
Comment (antho1404) [inline server/grpc/api/service.go]: this should also be in the toProto

**Comment by NicolasMahe** [comment]
@ilgooz please either do the modif in this PR

File: sdk/service/service.go (hunk 1 above)
Comment (ilgooz) [reply sdk/service/service.go]: done

File: server/grpc/api/service.go (hunk 2 above)
Comment (ilgooz) [reply server/grpc/api/service.go]: done

**Comment by ilgooz** [comment]
still blocking the next PR, please merge ASAP

**Pull request merged**